static/src/javascripts/projects/common/modules/identity/upsell/checkbox/Checkbox.js

```diff
@@ -8,20 +8,19 @@ type CheckboxHtmlProps = {
 
 type CheckboxProps = {
     title: string,
-    subtitle: ?string,
+    uniqueId: string,
     checkboxHtmlProps: CheckboxHtmlProps,
 };
 
-const Checkbox = ({ title, subtitle, checkboxHtmlProps }: CheckboxProps) => (
+const Checkbox = ({ title, uniqueId, checkboxHtmlProps }: CheckboxProps) => (
     <label
-        data-link-name={`upsell-consent : checkbox : ${title} : ${
+        data-link-name={`upsell-consent : checkbox : ${uniqueId} : ${
```

Comment:
It was a bit mad to have the full description (title) of the opt-out in the data link name, so using a unique ID instead.

Reply:
👍